img/p/ptv.vic.gov.au.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="75.58" height="43.33" xmlns:v="https://vecta.io/nano"><path d="M0 40.304c0 1.67 1.35 3.02 3.018 3.02a3.02 3.02 0 0 0 3.022-3.02v-6.269h5.025c6.305 0 11.246-4.822 11.246-10.975 0-3.036-1.091-5.815-3.07-7.827-2.089-2.12-4.96-3.241-8.309-3.241H3.018A3.02 3.02 0 0 0 0 15.014m16.271 8.046c0 3.172-2.303 5.473-5.475 5.473H6.04V17.497h4.58c4.166 0 5.651 2.873 5.651 5.563m15.121 17.244a3.02 3.02 0 1 0 6.04 0v-22.54h5.736a2.89 2.89 0 0 0 2.886-2.885c0-1.596-1.294-2.887-2.886-2.887H25.656c-1.593 0-2.886 1.291-2.886 2.887a2.89 2.89 0 0 0 2.886 2.885h5.736" fill="#231f20"/><path fill="#ec1d24" d="M73.379 18.235a3.66 3.66 0 0 0 2.201-3.356c0-1.504-.909-2.795-2.204-3.358L47.25.249c-1.53-.663-3.309.043-3.968 1.575a3.02 3.02 0 0 0 1.575 3.969l21.046 9.086-21.046 9.085a3.02 3.02 0 0 0-1.575 3.969 3.02 3.02 0 0 0 3.968 1.575"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 75.6 43.3"><path fill="#231f20" d="M0 40.3a3 3 0 0 0 3 3 3 3 0 0 0 3-3V34h5a11 11 0 0 0 11.3-11c0-3-1-5.8-3-7.8a12 12 0 0 0-8.4-3.2H3a3 3 0 0 0-3 3m16.3 8c0 3.2-2.3 5.5-5.5 5.5H6v-11h4.6c4.2 0 5.7 2.9 5.7 5.6m15 17.2a3 3 0 1 0 6.1 0V17.8h5.8a2.9 2.9 0 0 0 2.9-3c0-1.5-1.3-2.8-3-2.8H25.8a2.9 2.9 0 0 0-3 2.9 2.9 2.9 0 0 0 3 2.9h5.7"/><path fill="#ec1d24" d="M73.4 18.2a3.7 3.7 0 0 0 2.2-3.3 4 4 0 0 0-2.2-3.4L47.2.2a3 3 0 0 0-4 1.6 3 3 0 0 0 1.7 4l21 9-21 9.2a3 3 0 0 0-1.6 4 3 3 0 0 0 4 1.5"/></svg>
```

Reply:
Was there any reason to mark this as _resolved_ without committing the suggested change?